### ISSUE TITLE
Error running command, line 136 of 18.checkM_batch.pl

### ISSUE THREAD

**Issue by morien** [jtamames/SqueezeMeta#30]
I've got an error I can't quite figure out with checkM:

`Bin 132/356: metabat2.338.fa_sub.contigs.fa.tax
  Using profile for genus rank : Trachipleistophora
  Using profile for family rank : Pleistophoridae
  Using profile for  rank : Pansporoblastina
Error running command:    export PATH="/home/parfreylab/Desktop/lab_member_files/programs/SqueezeMeta/scripts/../bin/pplacer":$PATH; /home/parfreylab/Desktop/lab_member_files/programs/SqueezeMeta/scripts/../bin/checkm taxon_set  Pansporoblastina /home/parfreylab/Desktop/lab_member_files/morien/bat_guano_metagenomics/bat_guano/data/checkm_markers/Pansporoblastina.ms > /dev/null 2>&1 at /home/parfreylab/Desktop/lab_member_files/programs/SqueezeMeta/scripts/18.checkM_batch.pl line 136.`

There is an empty space in the command where the rank should be.

Here's the error if I run that command myself:
`checkm taxon_set: error: argument rank: invalid choice: 'Pansporoblastina' (choose from 'life', 'domain', 'phylum', 'class', 'order', 'family', 'genus', 'species')`

But, in fact, there is no such input file as Pansporoblastina.ms, so this may be the real problem?

Thanks in advance for your help, devs.

**Comment by morien** [comment]
Okay, did some digging on this today as I had more time. The taxonomy string had a rank name "subo", which was causing the issue. I looked at your code and saw the ranks are fed into the checkm command, and since suborder isn't a valid rank argument for checkm, the command failed. 

To fix, I went through all my .tax files for metabat and maxbin bins, and replaced the string "subo_" with "o_". Not the cleanest fix, but I thought I should report that changing that makes things work again.
